package.json

```diff
@@ -66,10 +66,11 @@
   },
   "overrides": {
     "es5-ext": "0.10.53",
-    "esbuild": "0.24.0",
+    "esbuild": "0.24.2",
     "formidable": "3.5.2",
     "got": "11.8.6",
     "react": "18.2.0",
+    "react-compiler-runtime": "latest",
```

Comment:
Explicit override to latest for `babel-preset-expo` and `graphiql` conflict on this dependency. `react-compiler-runtime` is currently using beta versions with commit hashes and both packages declare explicit versions of those beta versions which conflict. We should just use whatever is latest until it's out of beta.